pkg/synthetictests/networking.go

```diff
@@ -44,16 +47,27 @@ func testPodSandboxCreation(events monitorapi.Intervals) []*ginkgo.JUnitTestCase
 		}
 		deletionTime := getPodDeletionTime(eventsForPods[event.Locator], event.Locator)
 		if deletionTime == nil {
-			// this indicates a failure to create the sandbox that should not happen
-			failures = append(failures, fmt.Sprintf("%v - never deleted - %v", event.Locator, event.Message))
+			// look for network/machine-config progressing in operator events, stop when either found
+			operatorsProgressing := intervalcreation.IntervalsFromEvents_OperatorProgressing(events, event.From, event.To)
```

Comment:
So... is there a reason why you don't grab these out up top and prefilter by networking?  Then while it's O(N^2) it's at least O(MN) where M is small.

Reply:
Right, its easier to fetch all progressing operator intervals, filter by network - and then find if event is in the progressing networking via `.Slice()`. Implementing that in 41a2e0c6cb

Reply:
`sort.Search`/`Slice` was an incorrect approach, reworked this in 54660eec. See PR description for test result links, all have sandboxes errors marked as flakes as expected